fw/websocket.c

```diff
@@ -150,6 +150,7 @@ tfw_ws_srv_new_steal_sk(TfwSrvConn *srv_conn)
 	 */
 	conn->sk->sk_user_data = conn;
 
+	tfw_connection_put((TfwConn *)srv_conn);
```

Comment:
This is not looks correct for me. You do `tfw_connection_put` right before calling `destructor` manually, that looks strange. Also if `tfw_connection_put` will call `destructor` we will have problems with socket, it implies `srv_conn->sk = NULL;` must be placed before calling `tfw_connection_put`.